app/src/main/res/layout/nav_section_item.xml

```diff
@@ -1,16 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="16dp"
+    android:layout_height="wrap_content"
     android:background="@android:color/transparent"
     android:orientation="vertical"
     android:importantForAccessibility="no">
 
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:layout_centerVertical="true"
         android:background="?android:attr/listDivider"
         tools:background="@android:color/holo_red_dark"/>
-</RelativeLayout>
+
+    <TextView
+        android:id="@+id/nav_feeds_filtered_message"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="start"
+        android:paddingStart="@dimen/listitem_iconwithtext_textleftpadding"
```

Comment:
Not sure how it will look but maybe you could try aligning the padding of the TextView with the menu item titles.

Reply:
> Personally speaking, I like it more if the text is aligned with the subscriptions list, like it is now

I agree :+1: